shared/middleware/helpers/redis_table.rb

```diff
@@ -110,7 +119,7 @@ def table_from_row_key(key)
   # @param [String] key
   # @return [Integer] the row id.
   def id_from_row_key(key)
-    key.split('_')[1]
+    key.split('_')[1].to_i
```

Comment:
The existing usages were OK because it was passed into a function that accepted strings or integers and called to_i.  But I thought it was much better to call the to_i and match the docs!